addons/bestja_offers_categorization/templates.xml

```diff
@@ -2,15 +2,63 @@
     <data>
         <template id="list" name="Oferty">
             <t t-call="website.layout">
-                <h1 t-if="category">Kategoria "<t t-esc="category.display_name"/>"</h1>
-                <h1 t-if="not category">Wszystkie oferty</h1>
-                <ul>
-                    <li t-foreach="offers" t-as="offer">
-                        <a t-attf-href="{{ offer.get_public_url() }}">
-                            <t t-esc="offer.display_name"/>
-                        </a>
-                    </li>
-                </ul>
+                
+                <section class="oe_snippet_body">
+                    <div class="container-fluid">
+                        <div class="row">
+                            <div class="col-md-12">
+                                <h1 class="text-center mt48 mb48 section_title" t-if="category">Kategoria "<t t-esc="category.display_name"/>"</h1>
+                                <h1 class="text-center mt48 mb48 section_title" t-if="not category">Wszystkie oferty</h1>
+                            </div>
+                        </div>
+                    </div>
+                </section>
```

Comment:
Maybe it would be worth adding a message for cases when there are no offers in a particular category? (`t-if="not offers"`).